### ISSUE TITLE
PCC drop in openposev2 - pcc=0.9180459677147291

### ISSUE THREAD

**Issue by kamalrajkannan78** [tenstorrent/tt-xla#2938]
### Describe the bug

- Openposev2 model having `PCC=0.9180459677147291` - [jan23_openpose.log](https://github.com/user-attachments/files/24814606/jan23_openpose.log)
- Drop originates from this [Conv2d + BatchNorm2d](https://github.com/osmr/pytorchcv/blob/7f31dc8fe521e99012ac863a946e10a314118c16/pytorchcv/models/common/conv.py#L281C1-L283C27) present in `decoder.unit1.body.block1.body.block1` - [arch.txt](https://github.com/user-attachments/files/24796287/arch.txt)
- Ops passing with valid PCC on individual runs. 
- Below experiments proves above statements
  - Case 1 : Conv2d + BatchNorm2d - `PCC=0.9627268164615045` - [jan23_openpose_case1.log](https://github.com/user-attachments/files/24814609/jan23_openpose_case1.log)
  - Case 2 : Conv2d - `PCC=1.0`- [jan23_openpose_case2.log](https://github.com/user-attachments/files/24814616/jan23_openpose_case2.log)
  - Case 3 : BatchNorm2d  - `PCC=1.0` - [jan23_openpose_case3.log](https://github.com/user-attachments/files/24814622/jan23_openpose_case3.log)

  
### Repro

```
git checkout kkannan/jan23_openpose_pcc_drop
git submodule update --init --recursive
git lfs pull --include '*.pt’

# place the contents of attached loader scripts in model loader and run below command for model & exp runs
pytest tests/runner/test_models.py::test_all_models_torch["openpose/v2/pytorch-single_device-inference"] -svv

# emitpy 
python examples/pytorch/codegen/conv2d_bn2d.py  
cd conv2d_bn2d_jan23/ 
./run
```

### TT-METAL Ticket

- Replicated the drop in tt-metal with Conv2d + BatchNorm2d - https://github.com/tenstorrent/tt-metal/issues/36394

### Logs

- [conv2d_bn2d_jan23.zip](https://github.com/user-attachments/files/24814637/conv2d_bn2d_jan23.zip)
- [jan23_conv2d_bn2d_emitpy_gen.log](https://github.com/user-attachments/files/24814639/jan23_conv2d_bn2d_emitpy_gen.log)
- [jan23_conv2d_bn2d_emitpy_run.log](https://github.com/user-attachments/files/24814638/jan23_conv2d_bn2d_emitpy_run.log)

### Needed scripts 

- [loader_full_model.py](https://github.com/user-attachments/files/24814645/loader.py)
- [loader_case1.py](https://github.com/user-attachments/files/24814650/loader.py)
- [loader_case2.py](https://github.com/user-attachments/files/24814654/loader.py)
- [loader_case3.py](https://github.com/user-attachments/files/24814666/loader.py)
